packages/fury-adapter-oas3-parser/lib/parser/oas/parseOpenAPIObject.js

```diff
@@ -25,6 +25,7 @@ const unsupportedKeys = ['servers', 'security', 'tags', 'externalDocs'];
  * @param member {MemberElement}
  * @returns {boolean}
  * @see unsupportedKeys
+ * @private
  */
 const isUnsupportedKey = R.anyPass(R.map(hasKey, unsupportedKeys));
```

Comment:
Would it make sense for tacit (point-free) functions like these to add `@sig` containing Hindley-Millner type signature ? Functional programmers would welcome it when reading the code. Example of using it in jsdoc: https://github.com/char0n/ramda-adjunct/blob/master/src/allEqual.js#L16

How to write signatures: https://github.com/ramda/ramda/wiki/Type-Signatures